tests/dune

```diff
@@ -1,6 +1,7 @@
 (library
  (name util)
  (modules util)
+ (flags -w +3)
```

Comment:
- isn't warning 3 turned on by default already?
- this call is dropping other flags, it should read `(flags :standard -w .. )`

Reply:
Yes, you are right. A better fix of the compilation problem is and [@warning] that is in commit [4690761](https://github.com/savonet/ocaml-ssl/pull/104/commits/46907610c2b908368a9d8f8eb9e592f0f660e593)